moveit_ros/moveit_servo/src/servo_node.cpp

```diff
@@ -303,6 +303,8 @@ void ServoNode::servoLoop()
   }
   KinematicState current_state = servo_->getCurrentRobotState();
   last_commanded_state_ = current_state;
+  // Ensure the filter is up to date
+  servo_->resetSmoothing(current_state);
```

Comment:
Now the initial joint state is retrieved here. A few lines above, we wait for the up-to-date robot state.